src/pages/_app.tsx

```diff
@@ -62,7 +66,7 @@ function MyApp({ Component, pageProps }) {
                 )}
             </Head>
             <Provider store={store}>
-                <ThemeProvider theme={materialTheme}>
+                <ThemeProvider theme={AletheiaThemeConfig(safeNamespace)}>
```

Comment:
suggestion: `AletheiaThemeConfig()` is called on every render, creating a new theme object each time. This defeats React's memoization and causes unnecessary re-renders throughout the entire component tree. As a suggestion we could memoize the theme object